_posts/2020-3-27-SciML.md

```diff
@@ -0,0 +1,392 @@
+have previously demonstrated this capability with tools like [diffeqpy](https://github.com/SciML/diffeqpy)
+and [diffeqr](https://cran.r-project.org/web/packages/diffeqr/index.html) for
+using DifferentialEquations.jl from Python and R respectively, and we plan to
+continue along these lines to allow as much of our tooling as possible be accessible
+from as many languages as possible. While there will always be some optimizations
+that can only occur when used from the Julia programming language, DSL builders
+like [ModelingToolkit.jl](https://github.com/SciML/ModelingToolkit.jl) will be
+used to further expand the capabilities and performance of our wrappers.
+
+Here's an example which solves stochastic differential equations with high order adaptive methods
+
+from Python:
+
+```py
+# pip install diffeqpy
+from diffeqpy import de
+# diffeqpy.install()
+
+def f(du,u,p,t):
+    x, y, z = u
+    sigma, rho, beta = p
+    du[0] = sigma * (y - x)
+    du[1] = x * (rho - z) - y
+    du[2] = x * y - beta * z
+
+def g(du,u,p,t):
+    du[0] = 0.3*u[0]
+    du[1] = 0.3*u[1]
+    du[2] = 0.3*u[2]
+
+numba_f = numba.jit(f)
+numba_g = numba.jit(g)
+u0 = [1.0,0.0,0.0]
+tspan = (0., 100.)
+p = [10.0,28.0,2.66]
+prob = de.SDEProblem(numba_f, numba_g, u0, tspan, p)
+sol = de.solve(prob)
+
+# Now let's draw a phase plot
+
+ut = numpy.transpose(sol.u)
+from mpl_toolkits.mplot3d import Axes3D
+fig = plt.figure()
+ax = fig.add_subplot(111, projection='3d')
+ax.plot(ut[0,:],ut[1,:],ut[2,:])
+plt.show()
+```
+
+### We provide tools for researching methods in scientific machine learning
+
+Last but not least, we support the research activities of practitioners in
+scientific machine learning. Tools like [DiffEqDevTools.jl](https://github.com/SciML/DiffEqDevTools.jl)
+and [RootedTrees.jl](https://github.com/SciML/RootedTrees.jl) make it easy to
+create and benchmark new methods and accelerate the publication
+process for numerical researchers. Our wrappers for external tools like
+[FEniCS.jl](https://github.com/SciML/FEniCS.jl) and
+[SciPyDiffEq.jl](https://github.com/SciML/SciPyDiffEq.jl) make it easy to perform
+cross-platform comparisons. Our stack is entirely written within Julia,
+which means every piece can be tweaked on the fly, making it easy to mix
+and match Hamiltonian integrators with neural networks to discover new scientific
+applications. Our issues and [chat channel](https://gitter.im/JuliaDiffEq/Lobby)
+serve as places to not just debug existing software, but also discuss new
+methods and help create high performance implementations.
+
+In addition, we support many student activities to bring new researchers into the
+community. Many of the maintainers of our packages, like Yingbo Ma, Vaibhav Dixit,
+Kanav Gupta, Kirill Zubov, etc. all started as one of our over 50 student developers
+from past [Google Summer of Code](https://summerofcode.withgoogle.com/) and other
+[Julia Seasons of Contributions](https://julialang.org/jsoc/).
+

```

Comment:
Maybe mention that the fact that everything is written in julia makes it easier for students to 'look under the hood' and start understanding and tweaking libraries without learning 5 different languages.